includes/admin/views/html-admin-page-entries-view.php

```diff
@@ -163,6 +163,14 @@
 									</p>
 								<?php endif; ?>
 
+								<?php if ( ! empty( $entry->referer ) ) : ?>
+									<p class="everest-forms-entry-referer">
+										<span class="dashicons dashicons-admin-links"></span>
+										<?php esc_html_e( 'Referer Link:', 'everest-forms' ); ?>
+										<strong><a href="<?php echo esc_html( $entry->referer); ?>" target="_blank">View</a></strong>
```

Comment:
@coderkoala I guess $entry->referer is a URL so instead of using esc_html(), I guess esc_url() would be more appropriate. 
Please refer this for more detail on escaping functions https://developer.wordpress.org/themes/theme-security/data-sanitization-escaping/#escaping-securing-output

Reply:
@rabinstha daju, resolved with  074b827. Thanks a lot, I'll keep that in mind moving forward sanitizing urls :)

Reply:
@coderkoala Localisation is missing, naked word can't be used so please change use `esc_html_e( 'View', 'everest-forms' );`

Reply:
I've implemented your advise upon the code, and now it echos 'view' with localization. Thanks for catching that @shivapoudel daju.